### PR TITLE
systemvm: Update ISO URLs to the latest

### PR DESCRIPTION
**Pull request by khos2ow**
## Description

Update ISO URLs to the latest as the old URLs are borken.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Testing
- [ ] I have added tests to cover my changes.
- [ ] All relevant new and existing integration tests have passed.
- [ ] A full integration testsuite with all test that can run on my environment has passed.

### CODE REVIEW

**Comment by borisstoyanov** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Changes are only in systemvmtemplate appliance build code. I'll merge this based on reviews.

File: tools/appliance/builtin/template.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
       "disk_interface": "virtio",
       "net_device": "virtio-net",
 
-      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1708.iso",
-      "iso_checksum": "5848f2fd31c7acf3811ad88eaca6f4aa",
+      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1808.iso",
```
Comment (rhtyd) [inline tools/appliance/builtin/template.json]: I did not check this, looks like I made a mistake in merging this @khos2ow can you send a new PR to fix the iso_url, it ends with `1804` and not 1808.

Comment (khos2ow) [reply tools/appliance/builtin/template.json]: oh yes! copy paste error!

Comment (khos2ow) [reply tools/appliance/builtin/template.json]: the checksum is _correct_ and based on the official repo sha1 hash, but either the type needs to be changed from `md5` to `sha1` or manually calculate the checksum based on `md5`. We can follow the discussion in #2775

**Pull request merged**